gio-subclass/tests/simple_application.rs

```diff
@@ -81,18 +81,38 @@ mod imp {
 
 
         fn local_command_line(&self, application: &Application, arguments: &mut ArgumentList) -> Option<i32> {
-            // application.parent_local_command_line(arguments)
 
-            for line in arguments.iter() {
+            let mut rm = Vec::new();
+
+            for (i, line) in arguments.iter().enumerate() {
                 if line.starts_with("--local-"){
-                    println!("handling argument {:?} locally\n", line);
+                    println!("handling argument {:?} locally", line);
+                    rm.push(i)
+                }else{
+                    println!("not handling argument {:?} locally", line);
+
                 }
             }
-            arguments.remove(0);
 
-            Some(0)
+            rm.reverse();
+
+            for i in rm.iter(){
+                arguments.remove(*i);
+            }
+
+            None
```

Comment:
Don't we have to call `application.parent_local_command_line()` here?

Reply:
Could be, but in the original C test they don't either.

Reply:
Ok, fine with me then :) I still don't fully understand how that virtual method is supposed to be used

Now for the test, can you remove the output to stdout and add assertions for their effects? Then I believe this is ready to be merged